packages/contentstack-config/src/commands/config/get/rate-limit.ts

```diff
@@ -0,0 +1,44 @@
+import { cliux, configHandler, messageHandler } from '@contentstack/cli-utilities';
+import { Command } from '@contentstack/cli-command';
+import { RateLimitConfig } from '../../../interfaces';
+
+export default class RateLimitGetCommand extends Command {
+  static description: string = messageHandler.parse('Get rate-limit of organizations');
+  static examples = ['$ csdx config:get:rate-limit'];
+
+  async run() {
+    try {
+      const rateLimit = configHandler.get('rateLimit') || {};
+      const rateLimitData: RateLimitConfig = rateLimit || {};
+
+      const tableData = Object.entries(rateLimitData).map(([org, limits]) => ({
```

Comment:
@harshithad0703 , pls check the shared example
https://github.com/contentstack/cli/pull/1543#discussion_r1741906689